osf/external/gravy_valet/translations.py

```diff
@@ -84,6 +88,9 @@ def configured(self):
             attribute_name='credentials_available'
         )
 
+    def before_page_load(self, *args, **kwargs):
```

Comment:
Thanks! These are the sorts of things we'll probably have to keep finding and adding as we get deeper into integration 😓